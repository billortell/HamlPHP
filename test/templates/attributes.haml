!!! 5
%html
  %body
    %div('data-url' => 'aasd')
      %p('class' => 'paragraph', 'style' => 'background: #fff') Hei
      %p(class => 'paragraph', style => 'background: #fff') Hei
      %p(class => paragraph, style => background: #fff; color: #000) Hei