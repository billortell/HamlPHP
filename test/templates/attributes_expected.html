<!DOCTYPE html>
<html>
  <body>
    <div data-url="aasd">
      <p class="paragraph" style="background: #fff">Hei</p>
      <p class="paragraph" style="background: #fff">Hei</p>
      <p class="paragraph" style="background: #fff; color: #000">Hei</p>
      <p class="paragraph">Hei</p>
    </div>
  </body>
</html>
